app/containers/MessageBox/index.js

```diff
@@ -797,13 +857,38 @@ class MessageBox extends Component {
 		}
 	}
 
+	changeComposerState = () => {
+		const { fullScreenUp } = this.state;
+
+		if (fullScreenUp) {
+			this.setState({ translateY: BOTTOM });
+		} else {
+			this.setState({ translateY: TOP });
+		}
+
+		this.setState({ fullScreenUp: !fullScreenUp });
```

Comment:
How about combining them all to simplify it by reducing the branching and lines of code?
```
		this.setState({ 
			translateY: this.state.fullScreenUp ? BOTTOM : TOP, 
			fullScreenUp: !fullScreenUp 
		});
```

Reply:
I agree.
It's a good practice to trigger the least re-renders as possible.